tests/potential/test_pair.py

```diff
@@ -3,25 +3,26 @@
 
 import numpy as np
 
-from relentless import core
-from relentless.potential import pair
-from relentless.potential import potential
+import relentless
 
 class test_LennardJones(unittest.TestCase):
-    """Unit tests for pair.LennardJones"""
+    """Unit tests for relentless.potential.pair.LennardJones"""
```

Comment:
For all of these tests, you can access the classes using the import structure that we want the user to call:
```py
relentless.potential.LennardJones
relentless.potential.PairPotential
```
The `PairParameters` class looks useful enough that you can also add it to `__all__` so that it can be accessed like `relentless.potential.PairParameters`.